qobuz_dl/metadata.py

```diff
@@ -138,9 +158,12 @@ def tag_flac(
 
     artist_ = d.get("performer", {}).get("name")  # TRACK ARTIST
     if istrack:
-        audio["ARTIST"] = artist_ or d["album"]["artist"]["name"]  # TRACK ARTIST
+        artists = [artist_ or d["album"]["artist"]["name"]]  # TRACK ARTIST
     else:
-        audio["ARTIST"] = artist_ or album["artist"]["name"]
+        artists = [artist_ or album["artist"]["name"]]
+
+    artists.extend(_get_featured_artists(d))
+    audio["ARTIST"] = artists
```

Comment:
Are lists supported?

Reply:
Yes. see: https://mutagen.readthedocs.io/en/latest/user/gettingstarted.html#multiple-values or https://xiph.org/vorbis/doc/v-comment.html#implications for an example in the FLAC tag spec

Reply:
Sadly there's no way to know the exact format of the performers string. We'll just be guessing and may end with broken tags at any time. 

I wouldn't risk such a vital tag as ARTIST. If you find any other FLAC tag to suit your PR let me know.

Reply:
Unfortunately, this is the nature of things when the world cannot decide on a standard way to tag music.